packages/server/src/lib/email.js

```diff
@@ -17,8 +15,8 @@ function sendPasscode(email, passcode, httpOrigin) {
 function sendWelcomeEmail(email, httpOrigin) {
     return getTransport().send({
         toAddress: email,
-        subject: `${stateName} Grant Notification and Coordination Tool Access Link`,
-        body: `<p>You have been granted access to the CARES Grants:
+        subject: 'USDR Grants Tool Access Link',
```

Comment:
nit: in this email the link isn't a login link, I think -- it's just a link to the homepage. So having the subject line mention access link is a little confusing?

Maybe this one should say `Welcome to USDR Grants Tool`?

Reply:
Good catch, thanks Matt!

For the welcome email let's say:

Subject: `Welcome to USDR Grants Tool`
Body: `Visit USDR's Grants Tool at:`